docs/reference/cluster/nodes-stats.asciidoc

```diff
@@ -971,6 +974,29 @@ Contains statistics about all shards assigned to the node.
 (integer)
 The total number of shards assigned to the node.
 =======
+
+`mappings`::
+(object)
+Contains statistics about the mappings for the node.
+This is not shown for the `shards` level, since mappings may be
+shared across the shards of an index on a node.
++
+.Properties of `mappings`
+[%collapsible%open]
+=======
+`total_count`::
+(integer)
+Number of mappings.
```

Comment:
AIUI there are some subtleties around runtime fields and object fields and so on. If so, could we describe those subtleties here?

Reply:
@DaveCTurner I included that this includes object and runtime fields. What other subtleties would you have in mind to describe here?

Reply:
That'll do I think :)